staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5147,6 +5147,8 @@ const (
 	NodePIDPressure NodeConditionType = "PIDPressure"
 	// NodeNetworkUnavailable means that network for the node is not correctly configured.
 	NodeNetworkUnavailable NodeConditionType = "NetworkUnavailable"
+	// NodeCloudProviderUninitialized means that cloud provider has not initialized the node.
+	NodeCloudProviderUninitialized NodeConditionType = "Uninitialized"
```

Comment:
The field is called `NodeCloudProviderUninitialized`, but the actual condition is called `Uninitialized`.

Is this specific to cloud provider initialization or node initialization in general?

Reply:
since this is proposing mapping between the condition and an existing cloud-provider-specific taint, it is cloud-provider-specific, so the name should also be more specific